src/ckeditor.vue

```diff
@@ -0,0 +1,196 @@
+<template>
+	<component :is="tagName" ref="element" />
+</template>
+
+<script
+	setup
+	lang="ts"
+	generic="TEditor extends { create( ...args: any[] ): Promise<Editor> }"
+>
+import { debounce } from 'lodash-es';
+import {
+	ref,
+	watch,
+	markRaw,
+	onMounted,
+	onBeforeUnmount
+} from 'vue';
+import type { Editor, EditorConfig, EventInfo } from 'ckeditor5';
+
+type EditorType = TEditor extends { create( ...args: any[] ): Promise<infer E> } ? E : never;
+
+defineOptions( {
+	name: 'Ckeditor'
```

Comment:
To match the convention:

```suggestion
	name: 'CKEditor'
```

Reply:
The name of the global component is `Ckeditor` and the old name for this component was also `Ckeditor`, so I used this name for consistency.

https://github.com/ckeditor/ckeditor5-vue/blob/master/src/plugin.ts#L26
https://github.com/ckeditor/ckeditor5-vue/blob/master/src/ckeditor.ts#L21

AFAIK this name is only used for debugging / Vue devtools, but would using `CKEditor` be a breaking change?

Reply:
OK, it makes sense, so let's keep it as it is now.